I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/game_class.rpy", line 1, in script
    init python:
  File "game/game_class.rpy", line 1, in script
    init python:
  File "game/game_class.rpy", line 79, in <module>
    game.grid[y].append( Square(x=x, y=y, type = settings["tilemap"][y][x] ) )
  File "game/game_square_class.rpy", line 15, in __init__
    self.img.idle = tile.idle
AttributeError: 'Tiletype' object has no attribute 'idle'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 560, in main
    renpy.game.context().run(node)
  File "game/game_class.rpy", line 1, in script
    init python:
  File "game/game_class.rpy", line 1, in script
    init python:
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/game_class.rpy", line 79, in <module>
    game.grid[y].append( Square(x=x, y=y, type = settings["tilemap"][y][x] ) )
  File "game/game_square_class.rpy", line 15, in __init__
    self.img.idle = tile.idle
AttributeError: 'Tiletype' object has no attribute 'idle'

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
RENPY 1.0
Sun Jan 30 12:35:01 2022
